cmd/hdfs/mv.go

```diff
@@ -5,6 +5,7 @@ import (
 	"path"
 
 	"github.com/colinmarc/hdfs"
+	"github.com/colinmarc/hdfs/protocol/hadoop_hdfs"
 )
 
 func mv(paths []string, force, treatDestAsFile bool) {
```

Comment:
I think the functionality of `mv` should change, too. On my machine, `mv` overwrites by default.